src/cli/cli_br.cpp

```diff
@@ -258,7 +258,20 @@ template <> otError Br::Process<Cmd("onlinkprefix")>(Arg aArgs[])
     otError    error = OT_ERROR_NONE;
     PrefixType outputPrefixTypes;
 
-    SuccessOrExit(error = ParsePrefixTypeArgs(aArgs, outputPrefixTypes));
+    error = ParsePrefixTypeArgs(aArgs, outputPrefixTypes);
+
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_TESTING_API_ENABLE
+    if (error == OT_ERROR_INVALID_ARGS)
```

Comment:
While I understand this command is just used for testing and don't even need README doc, handling it in the error case seems not quite good. The code would be hard to update in the future.

What about we handle it like this:
* We check if the first argument is 'test'. If it's 'test', then we shift the arguments by one position, and do the handling for the test API. Of course, we can wrap the test block by the macro.
* Otherwise we keep the current handling, parse prefix types, and continue the handling.

We can do this in a separate PR.

Reply:
Thanks! I like this suggestion. :).... I've updated the code based on ths. It now requires `test` as the first argument, making the fact that this command is intended for testing only even clearer and more explicit.